weepickle-tests/src/test/scala/com/rallyhealth/weepickle/v1/TestUtil.scala

```diff
@@ -59,6 +59,13 @@ class TestUtil[Api <: com.rallyhealth.weepickle.v1.Api](val api: Api) {
     }
   }
 
+  def rwNull[T: api.To: api.From](t: T, s: String) = {
+    intercept[Exception] {
+      FromJson(s).transform(api.to[T])
+      api.from[T].transform(t, ToJson.string)
+    }
+  }
```

Comment:
I see that this was relaxed from https://github.com/rallyhealth/weePickle/pull/55/files#diff-f94bae8c817f05636dca69fa1b3afe71fbd28f73176a2c529310917070306c3cR64-R71 so that only one of these has to throw. Did the stricter check fail?

Reply:
Looks like the actual behavior is closer to:
```
  def rwNull[T: api.To: api.From](t: T, s: String) = {
    intercept[Exception] {
      FromJson(s).transform(api.to[T])
    }
    api.from[T].transform(t, ToJson.string)
  }
```

Reply:
Right, because right now JsonGeneratorVisitor implements `visitNull` to write null. We could have it throw like
```
override def visitNull(): JsonGenerator = throw new Abort("got null in Scala value")
```
which would fail Scala values that contain `null`. Otherwise, I think we have to leave the constraint the way you wrote and allow null values to be written, but not read.